src/index.js

```diff
@@ -53,15 +54,19 @@ async function bundleMDX(
     name: 'inMemory',
     setup(build) {
       build.onResolve({filter: /.*/}, ({path: filePath, importer}) => {
-        if (filePath === entryPath) return {path: filePath}
+        if (filePath === entryPath) return {path: filePath, pluginData: {inMemory: true}}
 
         const modulePath = path.resolve(path.dirname(importer), filePath)
 
-        if (modulePath in absoluteFiles) return {path: modulePath}
+        if (modulePath in absoluteFiles) return {path: modulePath, pluginData: {inMemory: true}}
 
         for (const ext of ['.js', '.ts', '.jsx', '.tsx', '.json', '.mdx']) {
           const fullModulePath = `${modulePath}${ext}`
-          if (fullModulePath in absoluteFiles) return {path: fullModulePath}
+          if (fullModulePath in absoluteFiles) return {path: fullModulePath, pluginData: {inMemory: true}}
+        }
+
+        if (cwd) {
+          return {}
         }
```

Comment:
If you wanna 😅

```suggestion
        if (cwd) return {}
```

I'd like to see a comment above this block here because it's not immediately clear to me what returning an empty object does.